### DISCUSSION TITLE
Channel 100+ Wifi

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#693] [Feature Requests]
For me it is now perfectly working , especially with the Radiodroid Trick for Webradio.
The only thing I'm missing is to use 5 GHz Wlan Channels above 100 .

**Comment by R101G** [comment]
From what I have read, this is because the proprietary Broadcom driver for the Pi 4 ignores the country setting. Apparently the Pi 3 works OK, but I haven't confirmed that.

It all works fine with a USB wifi dongle.
